src/networks.js

```diff
@@ -2,6 +2,28 @@
 // Dogecoin BIP32 is a proposed standard: https://bitcointalk.org/index.php?topic=409731
 
 module.exports = {
+  bitcoingold: {
+    messagePrefix: '\x18Bitcoin Gold Signed Message:\n',
```

Comment:
`\x18` is the hex of 24, which is the compact encoding of `len("Bitcoin Signed Message:\n")`.

As we are using "Bitcoin Gold Signed Message:\n", it should be `0x1d` (29 characters).